pkg/controllers/capiinstaller/capi_installer_controller.go

```diff
@@ -329,19 +331,18 @@ func (r *CapiInstallerController) extractProviderComponents(cm corev1.ConfigMap)
 	// Certain provider components have drone/envsubst environment variables interpolated within the manifest.
 	// Substitute them with the value defined in the environment variable (see setFeatureGatesEnvVars()).
 	// If that's not set, fallback to the default value defined in the template.
-	components, err := envsubst.EvalEnv(cm.Data["components"])
+
+	yamlManifests, err := extractManifests(cm)
 	if err != nil {
-		return nil, fmt.Errorf("failed to substitute env vars in CAPI manifests: %w", err)
+		return nil, fmt.Errorf("failed to extract manifests from configMap: %w", err)
```

Comment:
Could you also throw in this line the `namespace/name` for this configmap for easier debugging?

Reply:
This already had the ConfigMap name up the error chain. I've added namespace to it.

Reply:
Ok makes sense, thanks